security/pkg/server/ca/authenticate/kube_jwt.go

```diff
@@ -83,3 +106,57 @@ func (a *KubeJWTAuthenticator) Authenticate(ctx context.Context) (*Caller, error
 		Identities: []string{fmt.Sprintf(identityTemplate, a.trustDomain, callerNamespace, callerServiceAccount)},
 	}, nil
 }
+
+func (a *KubeJWTAuthenticator) validateRemoteK8sJwt(targetJWT string) ([]string, error) {
+	tokenReview := &v1.TokenReview{}
+	if a.jwtPolicy == jwt.JWTPolicyThirdPartyJWT {
+		tokenReview.APIVersion = "authentication.k8s.io/v1"
+		tokenReview.Kind = "TokenReview"
+		tokenReview.Spec = v1.TokenReviewSpec{
+			Token:     targetJWT,
+			Audiences: []string{"istio-ca"},
```

Comment:
@myidpt can you confirm audience is always istio-ca?

Reply:
Yes, that's correct.

Reply:
donot hardcoded